### PR TITLE
Update Mapsequence.plot docstring

### LINKED ISSUE #4301: Mapsequence.plot docstring lacks information on plot_function signature
it should either link to the animator docs or copy the function signature information

### PR DESCRIPTION
**Pull request by chouhanaryan**
### Description
Updated Mapsequence.plot docstring with plot_function signature and return information.

Fixes #4301

### CODE REVIEW

File: changelog/4472.docfix.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Add information about plot_function signature and return type to
```
Comment (nabobalis) [inline changelog/4472.docfix.rst:1]:
```suggestion
Added information about the :meth:`~sunpy.map.mapsequence.plot` return type.
```

File: changelog/4472.docfix.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+:func:`~sunpy.map.mapsequence.plot`.
```
Comment (nabobalis) [inline changelog/4472.docfix.rst:2]:
```suggestion
```

File: sunpy/map/mapsequence.py [hunk 3]
```diff
@@ -274,9 +274,17 @@ def plot(self, axes=None, resample=None, annotate=True,
...
+            For more information see `sunpy.visualization.MapSequenceAnimator`.
+
+        Returns
+        -------
+        ani : `matplotlib.animation.FuncAnimation`
+            A FuncAnimation instance.
+
+        See Also
+        --------
+        sunpy.visualization.mapsequenceanimator.MapSequenceAnimator
```
Comment (nabobalis) [inline sunpy/map/mapsequence.py:287]:
```suggestion
        `sunpy.visualization.mapsequenceanimator.MapSequenceAnimator`
```

File: sunpy/map/mapsequence.py [hunk 4]
```diff
@@ -274,9 +274,17 @@ def plot(self, axes=None, resample=None, annotate=True,
...
         plot_function : function
-            A function to be called as each map is plotted. Any variables
-            returned from the function will have their ``remove()`` method called
-            at the start of the next frame so that they are removed from the plot.
+            A function to be called as each map is plotted.
+            For more information see `sunpy.visualization.MapSequenceAnimator`.
+
+        Returns
+        -------
+        ani : `matplotlib.animation.FuncAnimation`
```
Comment (nabobalis) [inline sunpy/map/mapsequence.py:282]:
```suggestion
        `matplotlib.animation.FuncAnimation`
```

File: sunpy/map/mapsequence.py [hunk 5]
```diff
@@ -274,9 +274,17 @@ def plot(self, axes=None, resample=None, annotate=True,
...
-            A function to be called as each map is plotted. Any variables
-            returned from the function will have their ``remove()`` method called
-            at the start of the next frame so that they are removed from the plot.
+            A function to be called as each map is plotted.
+            For more information see `sunpy.visualization.MapSequenceAnimator`.
+
+        Returns
+        -------
+        `matplotlib.animation.FuncAnimation`
+            A FuncAnimation instance.
```
Comment (dstansby) [inline sunpy/map/mapsequence.py:283]: This line can be removed, as it's clear what the return type is from the above line.

**Review by dstansby — APPROVED** [review]
Looks 👍 to me, just one small optional suggestion.

File: sunpy/map/mapsequence.py (hunk 5 above)
Comment (nabobalis) [reply sunpy/map/mapsequence.py:283]: This is true but I kind of like the return section.

**Comment by nabobalis** [comment]
Failures are unrelated to the changes here.

**Comment by nabobalis** [comment]
Thanks @chouhanaryan

**Pull request merged**